specification/redhatopenshift/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the Azure Red Hat OpenShift API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2023-07
+tag: package-2023-09
 ```
 
 
+### Tag: package-2023-09
```

Comment:
We should add a net new block for this version and the tag should include the full date i.e.: package-2023-09-04